lib/spack/docs/getting_started.rst

```diff
@@ -941,45 +932,6 @@ other programs will also not work, because they also rely on OpenSSL.
 Once ``curl`` has been installed, you can similarly install the others.
 
 
-.. _InstallEnvironmentModules:
```

Comment:
This section is still relevant to using the module files that we generate, even if they're not necessary for `spack load` to work anymore.

Reply:
I don't think it belongs here. This section seems to be specifically focused on software you need to have for Spack to be fully functional. Anyhow, reviewing docs I found [this part](https://spack.readthedocs.io/en/latest/module_file_support.html#using-module-files-via-spack) that I didn't catch looking for mentions of `spack bootstrap`. This also needs to be revised, so maybe some brief information on how you could install a module system can be added there? In case would you prefer to have the modifications done in this PR or a later one?